README.md

```diff
@@ -22,7 +20,28 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+```ruby
+require 'simple_parameter_store'
+
+parameters = SimpleParameterStore.new(
+  client: Aws::SSM::Client.new,                           # optional, default: `Aws::SSM::Client.new`, can be used to set custom args for the SSM client
+  prefix: "/#{ENV['ENVIRONMENT']}",                       # optional, default: `nil`, can be used to prefix all parameter names with `/production`
+  expires_after: 3600,                                    # optional, default: `nil`, time in seconds after the store will be refreshed
+  decryp: true,                                           # optional, default: `true`, enable/disable automatic parameter decryption
+  names: {                                                # requires, hash with mapping of parameter names, the key will be used for the store index
+    foo: '/bar',                                          # aliased the key `/bar` (if prefix is `nil`) under the `:foo` in the store
+    max: ['max', :to_i.to_proc],                          # the value can be an array with the key as first and a caster as second value,
```

Comment:
I think the concept of casting could be explained a bit further (but not here in-place, I guess).